### PR TITLE
Release 2.3.5

### LINKED ISSUE #11525: BABEL_PARSER_SYNTAX_ERROR error with 2.3.1 when using d3
Getting BABEL_PARSER_SYNTAX_ERROR when starting up meteor on 2.2.1, 2.3 or 2.3.1.  Report here is for 2.3.1

Steps to reproduce:

1. create a todo app with blaze:
2.

### PR DESCRIPTION
**Pull request by StorytellerCZ**
Help us test the upcoming release:

```sh
meteor update --release 2.3.5-rc.0
```

This release includes:
* Updated Typescript #11566 
* Latest Babel to fix #11525 
* Node.js 14.17.5

### CODE REVIEW

**Comment by Gywem** [comment]
The issue with babel-parser is fixed in this PR, I just tested it. Thank you!

File: packages/ecmascript/transpilation-tests.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ Tinytest.add("ecmascript - transpilation - class methods", (test) => {
   test.isTrue(contains(output, 'Foo.staticMethod = function staticMethod('));
   test.isTrue(contains(output,
                        '.prototypeMethod = function prototypeMethod('));
-  test.isTrue(contains(output, '[computedMethod] = function ('));
+  test.isTrue(contains(output, '[_computedMethod] = function ('));
```
Comment (filipenevola) [inline packages/ecmascript/transpilation-tests.js:43]: Was this just breaking tests and not actual code?

Comment (StorytellerCZ) [reply packages/ecmascript/transpilation-tests.js:43]: As far as I can see yes. I will look if I can find something relevant in the changelogs, but from what I have been running the code and other tests there weren't any adverse effects.
:thinking: Not even as far as I have tested in production build.

**Comment by StorytellerCZ** [comment]
@Gywem rc.0 is out with new Node!

**Review by filipenevola — APPROVED** [review]

**Pull request merged**